### ISSUE TITLE
RootActivity.java not found in latest react-native-navigation

### ISSUE THREAD

**Issue by satishpalli** [wix/react-native-navigation#163]
Hi Team,

RootActivity.java not found in this path: 
react-native-navigation/android/app/src/main/java/com/reactnativenavigation/activities/RootActivity.java

in latest relase and while launching in Genymotion "Unfortunately app stoped " message is getting. can you please help.

**Comment by adamski** [comment]
I wonder it its the same issue as I'm having: 
```
Error:(68, 71) error: package com.reactnativenavigation.activities does not exist
```

**Comment by guyca** [comment]
@satishpalli We made a mistake and published version 2.0.0 as latest. I marked 1.0.30 back as latest so just `npm i react-native-navigation` to get it.

For now, I recommend you set `react-native-navigation` version in `package.json` to `^1.0.0` so you don't get any breaking versions.
